### PR TITLE
MBL-221 add NPCs to uptime scenario

### PR DESCRIPTION
**Pull request by joanxie878**
What this PR do?

 We want to ensure that we can add NPCs through worker flag, this scenario can report both the NPCs for each worker and the number of total NPCs.

Test Link:
 https://buildkite.com/improbable/unrealgdk-nfr/builds/4186 (2_1 zone for 100 NPCs)

Test Result:
image

### CODE REVIEW

**Comment by jayprobable** [comment]
This PR depends on https://github.com/spatialos/UnrealGDKTestGyms/pull/333#pullrequestreview-624904937 ?

File: Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h [hunk 1]
```diff
@@ -28,6 +28,7 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 			, ExpectedCount(InExpectedCount)
 			, Variance(InVariance)
 		{}
+		explicit FExpectedActorCount() {}
```
Comment (jeffxu-improbable) [inline Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h:31]:
```suggestion
		explicit FExpectedActorCount(): ExpectedCount(0), Variance(0) {}
```

File: Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp [hunk 2]
```diff
@@ -456,40 +458,20 @@ void ABenchmarkGymGameModeBase::TickActorCountCheck(float DeltaSeconds)
...
-						*NFRFailureString,
-						*ExpectedActorCount.ActorClass->GetName(),
-						ExpectedCount,
-						ActualCount);
-				}
-				break;
-			}
-		}
+		const FString WorkerID = FPlatformProcess::ComputerName();
+		const int32 ActualCountSimlate = GetActorClassCount(ExpectedSimPlayersCount.ActorClass);
```
Comment (jeffxu-improbable) [inline Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp:470]: `ActualCountSimulate` ?

**Review by jeffxu-improbable — APPROVED** [review]
I found some spell error, other parts are good!

**Review by jayprobable — APPROVED** [review]

**Pull request merged**